libs/h3/build.gradle

```diff
@@ -50,6 +50,16 @@ project.loggerUsageCheck.enabled = false
 project.forbiddenApis.ignoreFailures = true
 
 publishing {
+    repositories {
+        maven {
+            name = "Snapshots"
+            url = "https://aws.oss.sonatype.org/content/repositories/snapshots"
+            credentials {
+                username "$System.env.SONATYPE_USERNAME"
+                password "$System.env.SONATYPE_PASSWORD"
+            }
+        }
+    }
```

Comment:
Does this have any effect? Do we need it? Dont want publishing unwanted artifacts so just confirming.

Reply:
It shouldn't have impact as of now. We are not publishing as an independent plugin yet

Reply:
In that case can we remove it? Reason being a change in gradlew tasks can publish the artifacts since access is controlled at group level only. 
Thanks!

Reply:
Ack